src/utils/utils.cairo

```diff
@@ -16,11 +16,18 @@ from starkware.cairo.common.memcpy import memcpy
 from starkware.cairo.common.pow import pow
 from starkware.cairo.common.uint256 import (
     Uint256,
+    uint256_add,
     uint256_and,
     uint256_check,
+    uint256_mul_div_mod,
+    uint256_mul,
     uint256_eq,
+    uint256_lt,
     uint256_shr,
+    uint256_sub,
+    uint256_unsigned_div_rem,
 )
+
```

Comment:
unused imports ?

Reply:
in the end i kept them since i added gas cost calcuation again

Reply:
but they are not used in the file

Reply:
correct, those are for `modexp.utils`, will remove